pages/09.contacts/01.manage-contacts/02.contact-monitoring/docs.en.md

```diff
@@ -197,9 +202,22 @@ These are just a few of the integrations already created by the Mautic community
 
 ### Identify visitors by tracking URL
 
-Mautic 2.9 added an option to the configuration section for identifying visitors by tracking URL. If enabled, returning visitors will be identified by tracking URLs from channels (especially from emails) when no cookie exists yet.
+There is a configuration section for identifying visitors by tracking URL although this is not recommended for use because it could be used to spoof tracking. If enabled, returning visitors will be identified by tracking URLs from channels (especially from emails) when no cookie exists yet.
+
+Note: The email contact field has to be marked as a unique identifier and publicly updatable in your Mautic configuration.
 
-Note: The email contact field has to be marked as a unique indentifier and publicly updatable in your Mautic configuration.
+
+#### How are contacts tracked with the tracking script?
+
+When using the tracking script, contacts are tracked with third party cookies on the Mautic instance's domain and/or the browser's local storage.
+
+Although the script will write first party cookies to the tracked domain (expires with the session), they are NOT used for tracking. See "Local contact cookie (first party cookie)."
+
+When a contact visits the website for the first time, the tracking script will make a call to Mautic. Mautic will check if `mautic_device_id` cookie is set on it's domain. If it exists and if the device_id is found in Mautic's database, Mautic will recognize the request as the contact associated with the given device.
+
+Mautic will return the contact ID, the device ID, and a legacy session ID (this is the same as the device ID). These value are stored in the browser's local storage (if applicable) and written to the site's domain as a first party cookie (not used for tracking).
+
+The next time the tracking script sends a request to Mautic, it will use the device ID from the browser's local storage to identify the tracked contact. If that cannot be found, Mautic will default to the cookies stored on it's own domain (so third party cookies) to identify the contact.
```

Comment:
```suggestion
The next time the tracking script sends a request to Mautic, it will use the device ID from the browser's local storage to identify the tracked Contact. If that cannot be found, Mautic will default to the cookies stored on it's own domain (so third party cookies) to identify the Contact.
```